content/en/docs/reference/flow-control/_index.md

```diff
@@ -0,0 +1,181 @@
+---
+title: Flow control
+weight: 120
+no_list: true
```

Comment:
Since this is a "no_list" subdirectory, maybe we can create a `reference/debug-cluster` directory and put this page under that directory. When we name this file as `reference/debug-cluster/flow-control.md`, we are making room for future topics related to debugging cluster.

Reply:
Moved to `reference/debug-cluster`. Please let me know if `weight` and other configuration make sense.